splittext.go

```diff
@@ -26,9 +26,20 @@ func (f *Fpdf) SplitText(txt string, w float64) (lines []string) {
 	i := 0
 	j := 0
 	l := 0
+	fontLength := len(cw)
```

Comment:
```suggestion
	sz := len(cw)
```
(that's not really the font length, is it ?)

Reply:
actually, I would just remove this local variable and use `len(cw)` in the `for-loop` below.

Reply:
I introduced the local var to skip calculating the array's length nb times but removing the local var is ok with me